commands/builtins/builtins.go

```diff
@@ -354,10 +351,7 @@ func (l lastCommand) Execute(job jobs.Job) (string, error) {
 	callingUser := job.Request.Username
 	jobs, err := jobs.Find(jobs.JobFilter{
 		Limit: 1,
-		Match: func(j jobs.Job) bool {
-			return j.Request.Username == callingUser &&
-				j.Request.Command != BuiltinLastCommand
```

Comment:
Should this be ported too?

Reply:
I think not simply because builtin and tail commands are not recorded, so they will never pop up.

Reply:
Oh, true. Then ignore.